aea/registries/base.py

```diff
@@ -695,25 +699,35 @@ def handle_internal_messages(self) -> None:
         :return: None
         """
         while not self.decision_maker_out_queue.empty():
-            tx_message = (
+            # TODO get_nowait in this context works only if the queue is accessed only here
```

Comment:
@marcofavorito this seems to be an edges edge case. Feel free to handle the exception by logging it.